src/main/pg/pg_ids.h

```diff
@@ -126,7 +126,7 @@
 #define PG_PINIOBOX_CONFIG 530
 #define PG_USB_CONFIG 531
 #define PG_SDIO_CONFIG 532
-#define PG_DISPLAY_PORT_CRSF_CONFIG 533
+//#define PG_DISPLAY_PORT_CRSF_CONFIG 533
```

Comment:
This was never released. I think we can just remove this group, and renumber `PG_TIMER_IO_CONFIG`.

Reply:
Will do